packages/extension-chakra-storefront/src/theme/index.js

```diff
@@ -16,89 +16,126 @@ import layerStyles from './foundations/layerStyles'
+// import NestedAccordion from './components/project/nested-accordion'
+// import LocaleSelector from './components/project/locale-selector'
+import offlineBannerRecipe from './components/project/offline-banner'
+// import Pagination from './components/project/pagination'
+// import ProductTile from './components/project/product-tile'
+// import SocialIcons from './components/project/social-icons'
+// import SwatchGroup from './components/project/swatch-group'
+// import ImageGallery from './components/project/image-gallery'
 
 // Please refer to the Chakra-Ui theme customization docs found
 // here https://chakra-ui.com/docs/theming/customize-theme to learn
 // more about extending and overriding themes for your project.
-export const overrides = {
-    styles,
-    layerStyles,
-    colors,
-    sizes,
-    space,
-    gradients,
-    shadows,
-    components: {
-        // base components
-        Accordion,
-        Alert,
-        Badge,
-        Button,
-        Checkbox,
-        Container,
-        Drawer,
-        FormLabel,
-        Icon,
-        Input,
-        Modal,
-        Popover,
-        Radio,
-        Select,
-        Skeleton,
-        Tooltip,
 
-        // project components
-        App,
-        Breadcrumb,
-        Header,
-        Footer,
-        CheckoutFooter,
-        LinksList,
-        ListMenu,
-        DrawerMenu,
-        NestedAccordion,
-        LocaleSelector,
-        OfflineBanner,
-        SocialIcons,
-        Pagination,
-        ProductTile,
-        SwatchGroup,
-        ImageGallery
-    }
-}
+export const overrides = defineConfig({
+    ...styles,
+    theme: {
+        layerStyles,
+        tokens: {
+            colors,
+            sizes,
+            space,
+            gradients,
+            shadows,
+            fonts: {
+                heading: `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"`,
+                body: `-apple-system, BlinkMacSystemFont, "Segoe UI", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol"`,
+                mono: `SFMono-Regular,Menlo,Monaco,Consolas,"Liberation Mono","Courier New",monospace`
+            }
+        },
+        recipes: {
+            // Built-in components
+            badge,
+            button,
+            container,
+            formLabel,
+            icon,
+            input,
+            modal,
+            radio,
+            select,
+            skeleton,
+            popover
+        },
+        slotRecipes: {
+            // Built-in components
+            alert,
+            accordion,
+            drawer,
+            checkbox,
+            tooltip,
 
-export default extendTheme(overrides)
+            // project components
+            app: appRecipe,
+            header: headerRecipe,
+            offlineBanner: offlineBannerRecipe
+        }
+        // components: {
```

Comment:
Why are these ones commented out?

Reply:
Because Chakra v3 does not use component for styling components, they are now under `recipes` or `slotRecipes`